### PR TITLE
Assignment Commit

### PR DESCRIPTION
**Pull request by NallamalliSaiChandrika**
ass3

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+                        Websites
+                    </h1>
+                </td>    
+            </tr>
+            
+        </table>
+
+    </header>
+
+    <div class='empty'>
```
Comment (sneha-gowda) [inline index.html]: why empty div? if it's for spacing use margin top.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title> Assignment3</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <table class='table-tag'>
+            <tr>
+                <td style="padding: 0 60px 0px 0px;">
+                    <img src='logo.png' height="80px" width="80px" >
```
Comment (sneha-gowda) [inline index.html]: include alt attribute in img tag

**Comment by NallamalliSaiChandrika** [comment]
Ok

**Comment by Abduliitr** [comment]
Completed correctly!

**Pull request merged**